docs/src/pages/configurations/default-config/index.md

```diff
@@ -38,7 +24,146 @@ So, you could use any babel plugins or presets that you have used in your projec
 ## Webpack
 
 We use Webpack to serve and load JavaScript modules for the web.
-We've added some Webpack loaders to bring some good defaults. (This setup is very close to what you get with the [Create React App](https://github.com/facebookincubator/create-react-app).)
+
+The webpack config [is configurable](/configurations/webpack), and the default can depend on which framework you're using and whether you've used a generator like [Create React App](https://github.com/facebookincubator/create-react-app) or Angular CLI etc.
+
+> We're trying to make storybook more zero-config over time, **help to hook into the config of generators is very welcome**.
+
+<details>
+  <summary>This is what the config for storybook looks like when using CRA in dev-mode:</summary>
+
+  ```js
+  {
+    mode: 'development',
```

Comment:
Duplicated??

Reply:
yes, didn't know a good way of having a SSOT for this.